source/hooks/useTouchActivity.ts

```diff
@@ -0,0 +1,41 @@
+import { useState } from 'react';
+import { PanResponder } from 'react-native';
+import useInterval from './useInterval';
+
+export default function useTouchActivity(intervalDelay, touching) {
+  const [latestTouchTime, setLatestTouchTime] = useState(Date.now());
+  const [isTouching, setIsTouching] = useState(touching);
```

Comment:
I don't like this name. isActive feels better, maybe? otherwise it seems like the user should be touching the screen for this to be true.